consensus/polybft/polybft.go

```diff
@@ -150,6 +150,24 @@ func GenesisPostHookFactory(config *chain.Chain, engineName string) func(txn *st
 			return errMissingBridgeConfig
 		}
 
+		proxyAddr := contracts.GetProxyImplementationMapping()
+
+		var burnContractAddress types.Address
+
+		var isBurnContractSet bool
+
+		if burnContractAddress, isBurnContractSet = getBurnContractAddress(config, polyBFTConfig); isBurnContractSet {
```

Comment:
Can we shorten this a bit (I'm writing by heart, haven't checked in the IDE):
```suggestion
		burnContractAddress, isBurnContractSet := getBurnContractAddress(config, polyBFTConfig)

		if isBurnContractSet {
```

Reply:
fix 60fb909